README.md

```diff
@@ -214,6 +214,19 @@ function ratchet {
 }
 ```
 
+#### Pre-commit
+
+Ratchet can be ran as a pre-commit hook.
+In your `.pre-commit-config.yaml`:
+
+```yaml
+repos:
+  - repo: https://github.com/sethvargo/ratchet
+    rev: main
```

Comment:
This ideally is a commit sha. A git tag revision is most common for docs, but that requires the tag to include this change, so this would need updated after the next release. `main`/HEAD might be the best value to start with, so this at least works until the next release.